rmgpy/data/thermo.py

```diff
@@ -1356,54 +1364,65 @@ def get_thermo_data(self, species, training_set=None):
-        the database of adsorption energies, and the desired surface.
-
-        If binding_energies are not provided, resets the values to those
-        of the Pt(111) default.
+        Sets and stores the atomic binding energies specified in the input file.
 
         Args:
             binding_energies (dict, optional): the desired binding energies with
                 elements as keys and binding energy/unit tuples as values
 
         Returns:
-            None, stores result in self.delta_atomic_adsorption_energy
+            None, stores result in self.binding_energies
         """
-        reference_binding_energies = {
-            'C': rmgpy.quantity.Energy(-7.025, 'eV/molecule'),
-            'H': rmgpy.quantity.Energy(-2.754, 'eV/molecule'),
-            'O': rmgpy.quantity.Energy(-3.811, 'eV/molecule'),
-            'N': rmgpy.quantity.Energy(-4.632, 'eV/molecule'),
-        }
+        metal_db = MetalDatabase()
+        metal_db.load(os.path.join(settings['database.directory'], 'surface'))
 
-        # Use Pt(111) reference if no binding energies are provided
         if binding_energies is None:
-            binding_energies = reference_binding_energies
+            # Use Pt(111) reference that surfaceThermoPt111 was calculated on if no binding energies are provided
+            binding_energies = metal_db.get_binding_energies('Pt111')
+        elif binding_energies is str:
+            # Want to load the binding energies from the database
+            binding_energies = metal_db.find_binding_energies(binding_energies)
 
-        self.delta_atomic_adsorption_energy = {
-            'C': rmgpy.quantity.Energy(0.0, 'eV/molecule'),
-            'H': rmgpy.quantity.Energy(0.0, 'eV/molecule'),
-            'O': rmgpy.quantity.Energy(0.0, 'eV/molecule'),
-            'N': rmgpy.quantity.Energy(0.0, 'eV/molecule'),
-        }
+        for element, energy in binding_energies.items():
+            binding_energies[element] = rmgpy.quantity.Energy(binding_energies[element])
 
-        for element, deltaEnergy in self.delta_atomic_adsorption_energy.items():
-            deltaEnergy.value_si = binding_energies[element].value_si - reference_binding_energies[element].value_si
+        self.binding_energies = binding_energies
 
-    def correct_binding_energy(self, thermo, species):
+    def correct_binding_energy(self, thermo, species, metal_to_scale_to=None, metal_to_scale_from=None):
         """
         Changes the provided thermo, by applying a linear scaling relation
         to correct the adsorption energy.
 
         :param thermo: starting thermo data
         :param species: the species (which is an adsorbate)
+        :param metal: a tuple of the metal you're scaling from and the metal you want to scale to
         :return: corrected thermo
         """
+        metal_db = MetalDatabase()
+        metal_db.load(os.path.join(settings['database.directory'], 'surface'))
```

Comment:
I only load it because I want to search the database for certain binding energies (either pt111 if it was none, or for some string that matches an entry)

Reply:
My point is that it's already been loaded. Can't you access the one in memory instead of loading from disk again?

Reply:
Yeah, maybe try making a `load_metal_database` method, and attach it as attr of Thermodatabase?